frontend/src/components/ShootTickets/Ticket.vue

```diff
@@ -40,13 +39,15 @@ import { mapState, mapGetters } from 'vuex'
 import TimeString from '@/components/TimeString'
 import TicketLabel from '@/components/ShootTickets/TicketLabel'
 import TicketComment from '@/components/ShootTickets/TicketComment'
+import ExternalLink from '@/components/ExternalLink.vue'
 import sanitizeUrl from '@/mixins/sanitizeUrl'
 
 export default {
   components: {
     TimeString,
     TicketLabel,
-    TicketComment
+    TicketComment,
+    ExternalLink
```

Comment:
remove mixin

Reply:
we still need it in this component as I did not touch the buttons. Do we also want to use the external link component for buttons?